front/src/modules/timesStops/TimesStopsOutput.tsx

```diff
@@ -57,9 +57,10 @@ const TimesStopsOutput = ({
         return cx({
           'warning-schedule': arrivalScheduleNotRespected,
           'warning-margin': negativeDiffMargins,
+          'cell-index': columnId === 'ch',
         });
       }}
-      headerRowHeight={65}
+      headerRowHeight={50}
```

Comment:
It's 40px on the mockup
```suggestion
      headerRowHeight={40}
```

Reply:
The issue is that in some languages we can get header names that are too long to fit on 2 lines. On the other hand, we could also make the columns wider to ensure they always fit on two lines regardless of the language. The mockup was based on the idea that the column width would adapt to the headers rather than be fixed.

If we do want to instead go for columns tightly fitted to the content size, I believe we may need to change the library we use for the tables.

Reply:
We now use abbreviations (with title tags for the full header text) for headers that are too long. The header height was thus adjusted back to 40px.

Reply:
Ok, thanks for the explanations :pray:

Reply:
Ah hum it's still 50px